metrics/web/test/common/presentation/navigation/route_configuration/metrics_page_route_configuration_factory_test.dart

```diff
@@ -0,0 +1,177 @@
+      () {
+        final page = createMetricsPage(routeName: loginRouteName);
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration, equals(DefaultRoutes.login));
+      },
+    );
+
+    test(
+      ".create() returns a route configuration with the parameters equal to the given page parameters if the page route name is a login",
+      () {
+        final page = createMetricsPage(
+          routeName: loginRouteName,
+          pageParameters: pageParameters,
+        );
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration.parameters, equals(pageParametersMap));
+      },
+    );
+
+    test(
+      ".create() returns a dashboard route configuration if the given page route name is a dashboard",
+      () {
+        final page = createMetricsPage(routeName: dashboardRouteName);
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration, equals(DefaultRoutes.dashboard));
+      },
+    );
+
+    test(
+      ".create() returns a route configuration with the parameters equal to the given page parameters if the page route name is a dashboard",
+      () {
+        final page = createMetricsPage(
+          routeName: dashboardRouteName,
+          pageParameters: pageParameters,
+        );
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration.parameters, equals(pageParametersMap));
+      },
+    );
+
+    test(
+      ".create() returns a project groups route configuration if the given page route name is a project groups",
+      () {
+        final page = createMetricsPage(routeName: projectGroupsRouteName);
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration, equals(DefaultRoutes.projectGroups));
+      },
+    );
+
+    test(
+      ".create() returns a route configuration with the parameters equal to the given page parameters if the page route name is project groups",
+      () {
+        final page = createMetricsPage(
+          routeName: projectGroupsRouteName,
+          pageParameters: pageParameters,
+        );
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration.parameters, equals(pageParametersMap));
+      },
+    );
+
+    test(
+      ".create() returns a debug menu route configuration if the given page route name is a debug menu",
+      () {
+        final page = createMetricsPage(routeName: debugMenuRouteName);
+
+        final configuration = pageRouteConfigurationFactory.create(page);
+
+        expect(configuration, equals(DefaultRoutes.debugMenu));
+      },
+    );
+
+    test(
+      ".create() returns a route configuration with the parameters equal to the given page parameters if the page route name is a debug menu",
```

Comment:
✅ 
It seems that we can do something like for the page factory. In this case, we always use the given page parameters except for the loading configuration (the loading config is never created with the page parameters). And there we had a path and route name we always used. Can we simplify these test cases as well?